tools/grammars/compiler/converter.go

```diff
@@ -75,13 +75,35 @@ func (conv *Converter) AddGrammar(source string) error {
 	repo.FixRules(knownScopes)
 
 	if len(repo.Errors) > 0 {
-		fmt.Fprintf(os.Stderr, "The new grammar %s contains %d errors:\n",
-			repo, len(repo.Errors))
-		for _, err := range repo.Errors {
-			fmt.Fprintf(os.Stderr, "    - %s\n", err)
+		// Split out warnings from errors
+		warnings := []error{}
+		errors := []error{}
+		for _, e := range repo.Errors {
+			switch e.(type) {
+			case *MissingIncludeError, *UnknownKeysError:
+				warnings = append(warnings, e)
+			default:
+				errors = append(errors, e)
+			}
+		}
+		if len(errors) > 0 {
+			fmt.Fprintf(os.Stderr, "The new grammar %s contains %d errors:\n",
```

Comment:
Move the important stuff to the beginning of the line, and the part the user already knows to the end:
```suggestion
			fmt.Fprintf(os.Stderr, "%d errors found in new grammar '%s'\n",
```

Reply:
Done